Xamarin.Essentials/Maps/Map.uwp.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Globalization;
+using System.Threading.Tasks;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Map
+    {
+        internal static Task PlatformOpenMapsAsync(double latitude, double longitude, MapLaunchOptions options)
+        {
+            var uri = CreatePointUri(latitude, longitude, options);
+            return LaunchUri(uri);
+        }
+
+        internal static Task PlatformOpenMapsAsync(Placemark placemark, MapLaunchOptions options)
+        {
+            var uri = CreatePlacemarkUri(placemark, options);
+            return LaunchUri(uri);
+        }
+
+        static async Task LaunchUri(Uri mapsUri)
+        {// Use launcher api once it is available
+            await Windows.System.Launcher.LaunchUriAsync(mapsUri);
+        }
+
+        static Uri CreatePlacemarkUri(Placemark placemark, MapLaunchOptions options)
```

Comment:
I would just bring these methods inline as they are not re-used and don't need to be their own method to be honest.

Reply:
Done